rpcs3/Emu/Cell/Modules/cellSaveData.cpp

```diff
@@ -645,7 +646,7 @@ static NEVER_INLINE error_code savedata_op(ppu_thread& ppu, u32 operation, u32 v
 				{
 					doneGet->sizeKB += static_cast<s32>(::align(file.size, 4096));
 
-					if (!fs::remove_file(del_path + file.name))
+					if (!vfs::host::unlink(del_path + file.name))
```

Comment:
Don't use this unlink function. There is no assumption that remove_file might try to delete a file in use.